internal/endpoints/awsrulesfn/partitions.go

```diff
@@ -0,0 +1,170 @@
+// Code generated by endpoint/awsrulesfn/internal/partition. DO NOT EDIT.
```

Comment:
I'm going to make my comments on this generated file rather than on the codegen which I think will be clearer (just letting you know I realize this is generated).